<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpToDateDialog</class>
 <widget class="QDialog" name="JumpToDateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>עבור לתאריך</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>עבור לתאריך</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QDateEdit" name="gDateDateEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>yy/MM/dd</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="hebDateRadioButton">
        <property name="text">
         <string>לתאריך עברי</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="hDayComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="hMontComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="hYearComboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="gDateRadioButton">
        <property name="text">
         <string>לתאריך לועזי</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
